<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>balanceDialog</class>
 <widget class="QWidget" name="balanceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>balanceDialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>471</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>青色</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="rSlider">
        <property name="minimum">
         <number>-255</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>红色</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>紫色</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="gSlider">
        <property name="minimum">
         <number>-255</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>绿色</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>黄色</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="bSlider">
        <property name="minimum">
         <number>-255</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>蓝色</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>balanceDialog</receiver>
   <slot>accepted()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>balanceDialog</receiver>
   <slot>rejected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>accepted()</slot>
  <slot>rejected()</slot>
 </slots>
</ui>
